go/rpcclientlib/obscuro_client.go

```diff
@@ -49,11 +49,11 @@ type Client interface {
 	Stop()
 
 	// SetViewingKey sets the current viewing key on the client to be used for all sensitive request decryption
-	SetViewingKey(viewingKey *ecies.PrivateKey, pubKeyBytes []byte)
+	SetViewingKey(viewingKeyAddr common.Address, viewingKey *ecies.PrivateKey)
 
 	// RegisterViewingKey takes a signature for the public key, it verifies the signed public key matches the currently set private viewing key
 	//	and then submits it to the enclave
-	RegisterViewingKey(signerAddr common.Address, signature []byte) error
+	RegisterViewingKey(signature []byte, viewingPubKeyBytes []byte) error
```

Comment:
@BedrockSquirrel Can we rename this to `RegisterViewingKeyWithEnclave`? It wasn't clear to me what the difference was with the above.